test/infrastructure/kind/mapper_test.go

```diff
@@ -68,16 +76,16 @@ func TestGetMapping(t *testing.T) {
 			name:       "In case of multiple matches for Kubernetes version, return the most recent kind mode",
 			k8sVersion: semver.MustParse("1.23.17"),
 			expectedMapping: Mapping{
-				Mode:  Mode0_20,
-				Image: "kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb",
+				Mode:  Mode0_21,
+				Image: "kindest/node:v1.23.17@sha256:fbb92ac580fce498473762419df27fa8664dbaa1c5a361b5957e123b4035bdcf",
 			},
 		},
 		{
 			name:       "No match Future version gets latest kind mode",
-			k8sVersion: semver.MustParse("1.27.99"),
+			k8sVersion: semver.MustParse("1.29.99"),
```

Comment:
I think we should not bump the version in this test all the time. WDYT?

Reply:
ah, good point yes -- rereading the description i think it is okay to leave on a previous minor versions